docs/user/openstack/README.md

```diff
@@ -81,13 +81,6 @@ Swift is required for installation as the user-data provided by OpenStack Metada
 
 Swift is also used as a backend for the OpenShift image registry, but at the time of installation only an empty container is created without loading any data. Later on, for the system to work properly, you need to have enough free space to store the container images.
 
-The user must have `swiftoperator` permissions and `temp-url` support must be enabled. As an OpenStack administrator:
-
-```sh
-openstack role add --user <user> --project <project> swiftoperator
```

Comment:
AFAIK we still need the `swiftoperator` role to be able to create containers.

Reply:
yep

Reply:
I don't think it exists for director-deployed RGW installs. In that case the user needs Member